modules/openapi-generator/src/main/resources/cpp-qt5-client/helpers-body.mustache

```diff
@@ -7,14 +7,29 @@
 namespace {{this}} {
 {{/cppNamespaceDeclarations}}
 
+struct {{prefix}}CustomDateTimeFormat{
+    bool setString = false;
```

Comment:
Since it is a boolean, you could use `isStringSet` and same for other flag below.

Reply:
yep sounds good, done